packages/edit-site/src/components/page-pages/index.js

```diff
@@ -212,7 +213,14 @@ export default function PagePages() {
 			{
 				header: __( 'Title' ),
 				id: 'title',
+				fieldType: 'string',
```

Comment:
Unlike the `enumeration` type, `string` would be a free entry field. Do we need some sort of basic validation and the ability to add more? I'm thinking about things such as the length of the title doesn't exceed what a given API supports, whether it needs to adhere to a certain format even if it's a string, etc.

Reply:
This was my thinking: do we delegate validation to the `edit` function (the default provided by the field type or the custom one provided by the field)? or do we need a schema/validation mechanism that works independently from the `edit`?

For example, what do we do with an `email` field? Is it of type `string` or `email`? If the consumer provides a custom `edit` function, checking that the string introduced is an email would be their responsibility. However, if we use a proper validation/schema API, it's dataviews' – even if it's using a custom `edit` function.

I feel we don't have enough field variety to tell yet, though that's a question I wanted to bring up for us to keep in mind.